README.md

```diff
@@ -1,11 +1,15 @@
-# NeonAI Template TTS Plugin  # TODO: Name
+# NeonAI Coqui AI TTS Plugin
 [Mycroft](https://mycroft-ai.gitbook.io/docs/mycroft-technologies/mycroft-core/plugins) compatible
-TTS Plugin for Template Text-to-Speech.  # TODO: Update Name
+TTS Plugin for Coqui AI Text-to-Speech.
 
 # Configuration:
 # TODO: Specify any optional or required configuration values
```

Comment:
Please remove this `TODO` line

Reply:
Done with https://github.com/NeonGeckoCom/neon-tts-plugin-coqui-ai/pull/1/commits/dcb26578a636fdb127d5ba0b9e9dbbf63036bfb6